docs/docs/faq.md

```diff
@@ -69,7 +31,7 @@ Nebari automatically creates and manages `.bashrc` and `.profile`, so if the int
 source ~/.bashrc
 ```
 
-You can use `.bashrc` on Nebari, but it's important to note that by default Nebari sources `.bash_profile`. You should double-check to source the `.bashrc` inside the `.bash_profile`. Also, note that if you set environment variables in this way, these variables aren't available inside the notebooks.
+You can use `.bashrc` on Nebari, but it's important to note that by default Nebari sources `.bash_profile`. You should double-check to source the `.bashrc` inside the `.bash_profile`. Also, note that if you set environment variables in this way, these variables aren't available inside the notebooks. Additionally, the VS Code terminal does _not_ source `.bash_profile` by default.
```

Comment:
What does it mean "You can use .bashrc on Nebari"?  It seems like you can't really use it for anything custom b/c you can't modify it.

Also, I would put "it's important to note that by default Nebari sources `.bash_profile` if it exists" to help make it clear that it doesn't exist by default.

Reply:
I opened an issue about making vs code act the same as jupyterlab - https://github.com/nebari-dev/nebari/issues/2612

Reply:
Perhaps we should include an editable .bash_profile by default as well with [ -f ~/.bashrc ]; then . ~/.bashrc; fi already in it for users.

Reply:
This is not true: `You can use `.bashrc` on Nebari, but it's important to note that by default Nebari sources `.bash_profile`.` Not sure how I missed that when I was adding the VS Code statement. You CANNOT use .bashrc in nebari.

Reply:
I'll fix that and make the changes you requested.

Reply:
You can use .bashrc if you don't have a .bash_profile

Reply:
Yes, we are saying the same thing. I just added the VSCode sentence to the end of the existing docs - which were incorrect. I went ahead and reworked that section.